starter_code/routes/index.js

```diff
@@ -1,9 +1,75 @@
+
+
 const express = require('express');
 const router  = express.Router();
 
+const Movies =require("../models/movie.js");
+
 /* GET home page */
 router.get('/', (req, res, next) => {
   res.render('index');
 });
 
+
+router.get("/movies",(req,res,next)=>{
+  Movies.find()
+  .then(movieResults => {
+    console.log("IN MOVIES GET")
+    res.locals.moviesArray = movieResults; 
+   // res.send(movieResults);
```

Comment:
I like how you tried to send the data to the page first so you know what data you have